src/components/ui/OakPrimaryInvertedButton/OakPrimaryInvertedButton.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ElementType } from "react";
+
+import {
+  InternalShadowRectButton,
+  InternalShadowRectButtonProps,
+} from "@/components/ui/InternalShadowRectButton";
+import { PolymorphicPropsWithoutRef } from "@/components/utils/polymorphic";
+
+export type OakPrimaryInvertedButtonProps = Omit<
+  InternalShadowRectButtonProps,
+  | "defaultBorderColor"
+  | "defaultBackground"
+  | "defaultTextColor"
+  | "hoverBackground"
+  | "hoverBorderColor"
+  | "hoverTextColor"
+  | "disabledBackground"
+  | "disabledBorderColor"
+  | "disabledTextColor"
+>;
+
+export const OakPrimaryInvertedButton = <C extends ElementType = "button">({
+  element,
+  ...rest
+}: OakPrimaryInvertedButtonProps & PolymorphicPropsWithoutRef<C>) => {
+  return (
+    <InternalShadowRectButton
+      element={element ?? "button"}
+      defaultBorderColor="bg-btn-secondary"
+      defaultBackground="bg-btn-secondary"
+      defaultTextColor="text-primary"
+      hoverBackground="bg-btn-secondary"
+      hoverBorderColor="bg-btn-secondary"
+      hoverTextColor="text-primary"
+      disabledBackground="bg-btn-secondary"
```

Comment:
Same here - why doesn't the background render ?

Reply:
the background is white, I can wrap in in a coloured box in Storybook to make it clear